utilities/app/src/test/java/utilities/blogChallengeTester/BlogChallengeTester.java

```diff
@@ -0,0 +1,60 @@
+package utilities.blogChallengeTester;
+
+import org.junit.Test;
+import utilities.FizzBuzzTree;
+import utilities.blogChallenges.InsertSort;
+import utilities.tree.BinarySearchTree;
+import utilities.tree.Node;
+import utilities.tree.Tree;
+import org.junit.*;
+import static org.junit.Assert.*;
+
+import java.sql.Array;
+import java.sql.SQLOutput;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class BlogChallengeTester {
+
+    @Test public void insertSortedArrayTester(){
```

Comment:
Great work with multiple tests, even threw in some negative numbers! Well done Matthew!